tests/puppeteer/campaigns/smoke/crawlingFO.js

```diff
@@ -0,0 +1,32 @@
+// Importing pages
+const PAGE = require('../../pages/FO/home');
+const PRODUCT_PAGE = require('../../pages/FO/product');
+const CART_PAGE = require('../../pages/FO/cart');
+const productData = require('../data/FO/product');
+const cartData = require('../data/FO/cart');
+
+let page;
+let HOME;
+let PRODUCT;
+let CART;
+
+// Init objects needed
+const init = async () => {
+  page = await global.browser.newPage();
+  HOME = await (new PAGE(page));
```

Comment:
```suggestion
  homePage = await (new HomePage(page));
```
Using the new conventions.